methods/coin_activation/batch_activate.json

```diff
@@ -0,0 +1,14 @@
+[
+
+    {"userpass":"$userpass","method":"electrum","coin":"TOKEL","servers":[{"url":"1.eu.tokel.electrum.dexstats.info:10077"}]},
+    {"userpass":"$userpass","method":"electrum","coin":"BCH","servers":[{"url":"electrum1.cipig.net:10055"},{"url":"electrum2.cipig.net:10055"}]},
```

Comment:
I'd say. with this method being in playground repo, we need to ensure all methods can be run as is in the playground.
Like, here: change password to default `testpsw`, ensure electrum links are wss, so they can be activated with wasm.

Reply:
done (at least for where it was `$userpass`). I'll make sure to clean up others as I go thru the list.